tools.csv

```diff
@@ -8,4 +8,4 @@ rClr,NA,NA
 rSharp,1.1.2,NA
 Qualification Runner,12.1.116,NA
 Qualification Framework,3.4,NA
-PK-Sim,12.1.222,https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/download/v12.1.222/pk-sim-portable-setup.12.1.222.zip
\ No newline at end of file
+PK-Sim,12.1.222,https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/download/v12.1.222/pk-sim-portable-setup.12.1.222.zip

```

Comment:
@pchelle That's insane: missing end of line in the last line of the CSV file has lead to empty OSP version in created reports (even though the PK-Sim version was correctly read from the CSV file and the report was successfully created)
<img width="1003" height="403" alt="grafik" src="https://github.com/user-attachments/assets/5a4f1b27-a1ef-4aa2-b76b-d87415e2715d" />

And also the tools table saved in the PR was lacking the last line for PK-Sim:

<img width="463" height="530" alt="grafik" src="https://github.com/user-attachments/assets/12ec9e03-6305-4e7a-8a02-3a95bff7dee5" />